core/src/service/test_utils.rs

```diff
@@ -27,113 +24,6 @@ use crate::repo::root_repo;
 use crate::repo::{account_repo, db_version_repo};
 use crate::service::{file_service, integrity_service, path_service, sync_service};
 
-pub enum Operation<'a> {
```

Comment:
? you've resolved conflicts improperly

Reply:
compare to master

Reply:
Yeah I did the magic tool thing from intellij and it screwed everything up.